src/components/TodoCreate.jsx

```diff
@@ -0,0 +1,96 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import { flexCenter, flexColumn } from 'styles/commonStyle';
+import { v4 as uuid } from 'uuid';
+
+function TodoCreate({ dispatch }) {
+  const [inputValue, setInputValue] = useState('');
+  const [isError, setIsError] = useState(false);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (!inputValue?.trim()) {
+      setIsError(true);
+      return;
+    }
+
+    dispatch({
+      type: 'ADD_TODO',
+      payload: {
+        id: uuid(),
+        text: inputValue,
+      },
+    });
+
+    setIsError(false);
+    setInputValue('');
+  };
+
+  const handleInputValueChange = (e) => {
+    setInputValue(e.target.value);
+  };
+
+  return (
+    <InputForm onSubmit={handleSubmit}>
+      <InputContainer>
+        <InputWrapper>
+          <input className="input" placeholder="할 일 입력" value={inputValue} onChange={handleInputValueChange} />
+          <button className="input-button">입력</button>
+        </InputWrapper>
+        <ErrorMessage $isError={isError}>할 일을 입력해주세요</ErrorMessage>
+      </InputContainer>
+    </InputForm>
+  );
+}
+
+export default TodoCreate;
+
+const InputForm = styled.form`
+  display: flex;
+  width: 100%;
+  height: 6rem;
+  ${flexCenter}
+`;
+
+const InputContainer = styled.div`
+  ${flexColumn}
+  height: 100%;
+  gap: 1rem;
+`;
+
+const InputWrapper = styled.div`
+  display: flex;
+  height: 100%;
+  gap: 2rem;
+
+  & {
+    font-size: 2.5rem;
+  }
+
+  .input {
+    width: 50rem;
+    padding: 1rem;
+    border-radius: 1rem;
+  }
```

Comment:
이런식으로 영어가 길게 입력되었을 때 깨지고 있어요!
`word-break: break-all`로 해결할 수 있답니당 <br> 
<img width="348" alt="스크린샷 2024-03-24 오전 1 06 32" src="https://github.com/CEOS-Developers/react-todo-19th/assets/117785708/a43c64f1-f9e9-41fb-b303-958c4aa032b3">

```suggestion
  .input {
    width: 50rem;
    padding: 1rem;
    border-radius: 1rem;
    word-break: break-all; 
  }
```